### ISSUE TITLE
508-defect-2 [COGNITION]: COVID Vaccine - Heading levels should only increase by one

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#17156]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I want to hear heading levels presented in order. I might be confused when the first heading announced is a "heading level one" and the next one is a "heading level three". It seems like important content could have been missed.

## Details

The introductory page of our COVID initiative has an alert box with an H3 that should be overridable to an H2. For more information on why this is an important change, please see the [heading should increase by one](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/common-issues/headings-should-increase-by-one.md) guidance document.

## Acceptance Criteria

- [ ] Heading is changed to an H2
- [ ] Axe-core browser plugin no longer warns on this heading level skip (please ignore the new error you might see about the global footer H4)
- [ ] No changes to the page visually

## Environment

* https://staging.va.gov/health-care/covid-19-vaccine/stay-informed

## Screenshots or Trace Logs

![Screen Shot 2020-12-09 at 9 08 24 AM](https://user-images.githubusercontent.com/934879/101648766-c6d19600-39ff-11eb-8a60-d7430b45935d.png)
